app/src/js/widgets/fields/fieldVideo.js

```diff
@@ -107,11 +107,13 @@
          */
         _update: function () {
             var self = this,
-                url = 'https://api.embed.ly/1/oembed',
+                url = bolt.conf('paths.async') + 'oembed/request',
```

Comment:
All of it is getting re-done … I just went with what is there … maybe tomorrow, but I think that will have to wait for 3.3 TBH